pkg/types/nutanix/platform.go

```diff
@@ -0,0 +1,49 @@
+package nutanix
+
+// Platform stores any global configuration used for Nutanix platforms.
+type Platform struct {
+	// PrismCentral is the domain name or IP address of the Prism Central.
+	PrismCentral string `json:"prismCentral"`
+
+	// Insecure disables certificate checking when connecting to Prism Central.
+	Insecure bool `json:"insecure"`
```

Comment:
No insecure connections.

Reply:
For context. This is still https. We just allow disabling cert checking due to self signed certs.

Reply:
@thunderboltsid we will just have to doc download/import of certs like here: https://docs.openshift.com/container-platform/4.9/installing/installing_vsphere/installing-vsphere-installer-provisioned.html#installation-adding-vcenter-root-certificates_installing-vsphere-installer-provisioned